fleet_vehicle_inspection/models/fleet_vehicle_inspection_item.py

```diff
@@ -10,3 +10,4 @@ class FleetVehicleInspectionItem(models.Model):
     _description = 'Fleet Vehicle Inspection Item'
 
     name = fields.Char(required=True)
+    instruction = fields.Text(string='Instruction', size=140)
```

Comment:
```suggestion
    instruction = fields.Text(string='Instruction')
```

Is there any particular reason to set a size?

Reply:
I thought about the size of the field per house of the kanban mobile but it is better not to limit it.